### PR TITLE
Add base Scss stack

### PR DESCRIPTION
**Pull request by robsterlini**

### CODE REVIEW

File: {{cookiecutter.repo_name}}/styles/main.scss [hunk 1]
```diff
@@ -1,18 +1,33 @@
 /**
- * {{cookiecutter.project_name}}
+ * Ethanol
```
Comment (theskumar) [inline {{cookiecutter.repo_name}}/styles/main.scss]: Minor thing here, we should probably keep the `{{cookiecutter.project_name` here. :)

File: {{cookiecutter.repo_name}}/styles/main.scss [hunk 2]
```diff
@@ -1,18 +1,33 @@
...
  * =======
- * (c) Fueled
+ * Joe Richardson <joe.richardson@fueled.com>
```
Comment (theskumar) [inline {{cookiecutter.repo_name}}/styles/main.scss]: We should get rid of this is hard coded developer name.

**Review by theskumar — CHANGES_REQUESTED** [review]
I found minor issues with hard-coded values, rest looks good to go in!! :)

**Comment by robsterlini** [comment]
@saurabh-fueled good spot! Changes made!

**Review by dope — APPROVED** [review]
LGTM 👍

**Review by theskumar — APPROVED** [review]

**Pull request merged**